src/commands/Minion/decant.ts

```diff
@@ -54,8 +54,13 @@ export default class extends BotCommand {
 
 		await msg.author.settings.update(UserSettings.Bank, newBank);
 
+		const leftOverMessage =
```

Comment:
i think this would look better if placed inside the if block that already determines if there are doses to give back.